
<launch>

  <node pkg="ROSARIA" type="RosAria" name="ROSARIA"/>

 <!-- joy node -->
   <node respawn="true" pkg="wiimote"  type="wiimote_node.py" name="pioneer_wii" output="screen"/>

 <!-- Axes -->
  <param name="axis_linear" value="1" type="double"/>
  <param name="axis_angular" value="2" type="double"/>
  <param name="scale_linear" value="0.2" type="double"/>
  <param name="scale_angular" value="0.5" type="double"/>

  <node pkg="pioneer_teleop" type="pioneer_teleop_acc" name="teleop" output="screen"/>

</launch>